src/test/scala/com/cognite/sdk/scala/common/DataPointsResourceBehaviors.scala

```diff
@@ -3,33 +3,33 @@ package com.cognite.sdk.scala.common
 import java.time.Instant
 
 import com.softwaremill.sttp.Id
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{FlatSpec, Matchers, OptionValues}
```

Comment:
Since this PR currently doesn't use `OptionValues` in `DataPointsTest` I'm not sure we're using it right. Maybe it's getting included from somewhere else, but we can forget about it for now. It's pretty new to me as well.